tests/foreman/api/test_syncplan.py

```diff
@@ -252,17 +257,10 @@ def setUpClass(cls):
 
         """
         cls.org_id = entities.Organization().create_json()['id']
-        cls.prod1_id = entities.Product(
-            organization=cls.org_id
-        ).create_json()['id']
-        cls.prod2_id = entities.Product(
-            organization=cls.org_id
-        ).create_json()['id']
```

Comment:
I've dropped this because of BZ 1199150. Re-using these products for multiple tests causes unnecessary test failures. `test_repeatedly_add_remove` targets this issue.